portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,23 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Leo's  Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
+      <h1>Leo's Portfolio</h1>
       <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
+      <p>Click here to get a random greeting:</p>
       <button onclick="addRandomGreeting()">Hello</button>
       <div id="greeting-container"></div>
+      <p>My full name is Leobardo but I usually go by Leo. I'm 20 years old
+          and I'm from Phoenix, Arizona. </p>
+      <h1 style="text-align:center">Learn more about me below!</h1>
+      <p>Click <a href="interests.html">here</a> to see some of my interests!</p>
```

Comment:
Links without / are relative links, meaning that the path they redirect to is based on the path to the current page, e.g. if you make a page /example/contact.html and add a link to index.html, it will try to route to /example/index.html not /index.html. I think it's generally preferable to use absolute paths when building links so that it's clear where a link will go.

(Generally in complex web applications with many pages, the web framework you use will provide some way to auto-generate an absolute path to a page based on the identity of the page you are linking to without having to worry about what the page is in practice, so if you end up with something like pathOf(Contact), meaning that if you move the Contact page, you don't have to worry about updating from /example/contact.html to /testing/contacts.html everywhere, since the pathOf would take care of that, but for this project since it's all basic HTML you don't get any of that.)

Reply:
Got it, will update to reflect absolute paths